solanaj/src/main/java/org/p2p/solanaj/kits/transaction/TransferDetails.java

```diff
@@ -1,24 +1,27 @@
 package org.p2p.solanaj.kits.transaction;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
 import java.math.BigInteger;
 import java.util.Map;
 
 public class TransferDetails extends TransactionDetails {
     private final String destination;
     private final String source;
+    @Nullable
     private final String authority;
+    @Nullable
```

Comment:
surprisingly, but our app was not crashing when  authority or mint came null from server! we were really lucky.